kafka-ui-react-app/src/components/Topics/Topic/Edit/__test__/topicParamsTransformer.spec.ts

```diff
@@ -0,0 +1,93 @@
+import topicParamsTransformer from 'components/Topics/Topic/Edit/topicParamsTransformer';
+import { DEFAULTS } from 'components/Topics/Topic/Edit/Edit';
+
+import { completedParams, topicWithInfo } from './fixtures';
+
+describe('topicParamsTransformer', () => {
+  it('topic not found', () => {
+    expect(topicParamsTransformer(undefined)).toEqual(DEFAULTS);
+  });
+
+  it('topic  found', () => {
+    expect(topicParamsTransformer(topicWithInfo)).toEqual(completedParams);
+  });
+  it('topic  partitions', () => {
+    expect(topicParamsTransformer(topicWithInfo).partitions).toEqual(
+      completedParams.partitions
+    );
+    expect(
+      typeof topicParamsTransformer({
+        ...topicWithInfo,
+        partitionCount: undefined,
+      }).partitions
+    ).toEqual('number');
+  });
+
+  it('topic  maxMessageBytes', () => {
```

Comment:
describe maxMessageBytes
 - it 
 - it convert to number
 - it returns default value when